packages/backend/src/App.ts

```diff
@@ -387,7 +387,7 @@ export default class App {
     }
 
     private initSlack() {
-        const slackService = new SlackService({
+        const slackBot = new SlackBot({
```

Comment:
Not a blocker, but would it make sense to move this to a method on `SlackIntegrationService`, or another similar service? e.g `getIntegrationService().createSlackBot(...)`

Reply:
The slack bot starts an "ExpressReceiver" in port 4352 so it should be initialized in the App class. But I agree that this is not obvious and can be improved.